samples/apps/research-assistant-react/README.md

```diff
@@ -1,48 +1,46 @@
-# ARA - AutoGen Research Assistant
+# AutoGen Research Assistant
 
-ARA is an Autogen-powered AI research assistant that can converse with you to help you conduct research, write and execute code, run saved skills, learn new skills by demonstration, and adapt in response to your interactions.
+AutoGen Research Assistant is an Autogen-powered AI research assistant that can converse with you to help you conduct research, write and execute code, run saved skills, learn new skills by demonstration, and adapt in response to your interactions.
 
 Project Structure:
 
-- _backend/_ code for the web api, served by FastAPI.
+- _autogenra/_ code for the backend classes and web api (FastAPI)
 - _frontend/_ code for the webui, built with Gatsby and Tailwind
 
 ## Getting Started
 
 Install requirements:
 
 ```bash
-cd backend
-pip install -r requirements.txt
+pip install -e .
 ```
 
-AutoGen requires access to an LLM. Please see the [AutoGen docs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints) on how to configure access to your LLM. We recommend setting the `OAI_CONFIG_LIST` environment variable to point to your LLM config file.
-
-Also, if you plan to use bing search, set the `BING_API_KEY`
+AutoGen requires access to an LLM. Please see the [AutoGen docs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints) on how to configure access to your LLM. We recommend setting up your `OPENAI_API_KEY` environment variable first and `OAI_CONFIG_LIST` environment variable to contain a list of OpenAI model configurations.
 
 ```bash
-export OAI_CONFIG_LIST=/path/to/OAI_CONFIG_LIST
-export BING_API_KEY=<your bing api key>
+export OPENAI_API_KEY=<your_api_key>
 ```
 
 ### Build and Run Frontend UI
 
+Note that your Gatsby requires node > 14.15.0 . You may need to [upgrade your node](https://stackoverflow.com/questions/10075990/upgrading-node-js-to-latest-version) version as needed.
+
 ```bash
+npm install --global yarn
```

Comment:
Does this require sudo? If so, should we warn the users? (Eg, by adding a comment in this script)